glidertest/tools.py

```diff
@@ -488,15 +488,21 @@ def plot_section_with_srss(ds, ax, sel_var='TEMP',start_time = '2023-09-06', end
     plt.colorbar(c, label=f'{sel_var} [{ds[sel_var].units}]')
     return ax 
  
-def check_temporal_drift(ds, ax1, ax2, var='DOXY'):
-    ax1.scatter(mdates.date2num(ds.TIME),ds[var], s=10)
-    ax1.xaxis.set_major_formatter(DateFormatter('%Y-%m-%d'))
-    ax1.set(ylim=(np.nanpercentile(ds[var], 0.01), np.nanpercentile(ds[var], 99.99)), ylabel=var)
+def check_temporal_drift(ds: pd.DataFrame, var: str,ax: plt.Axes = None, **kw: dict,)-> tuple({plt.Figure, plt.Axes}):

```

Comment:
conventionally, one would expect ds to be an xarray DataSet, we'd use df for a pandas DataFrame. Nice type hints though!